### ISSUE TITLE
Load config Regex issue.

### ISSUE THREAD

**Issue by LikeSundayLikeRain** [dfrankland/hyperterm-tab-icons#31]
https://github.com/dfrankland/hyper-tab-icons/blob/ef444af2624ba6b22d5afcd8f8fbc8a69f6b1757/src/lib/getIcon.js#L40-L41

Seems it should be **config.processNameRegex.source** and **config.processNameRegex.flags**

**Comment by dfrankland** [comment]
Thanks for the heads up! [I just pushed your fix and made a new release for this.](https://github.com/dfrankland/hyper-tab-icons/commit/72ea26321ce6b76ef69f6cb79ef49d991cfb04dc) Mind testing it out?

**Comment by LikeSundayLikeRain** [comment]
It's got fixed. But I think I got an another issue.

My custom Regexp defined at config file can't get loaded when I'm launching Hyper, but it works fine when I reload Hyper after it launches.

Looks like Hyper is rendering the plugins before the window loads. So when initiating hyper-tab-icons, **const config = getConfig()** seems load nothing because of window is not defined.

I'm using Hyper 1.4.8 on a Windows machine, not sure if Hyper loads plugin differently on other systems.
